R/prepare_weo_2024_ecb_scenario.R

```diff
@@ -6,35 +6,40 @@
 #'   `WEO2024_Extended_Data_World.csv` import.
 #' @param weo_2024_fig_chptr_3_raw A tidyxl data frame containing a raw import
 #'   of `WEO2024_Figures_Chapter_03.xlsx`.
-#' @param iea_global_ev_raw A data frame containing a raw `IEA Global EV Data
-#'   2024.csv` import.
+#' @param iea_global_ev_2024_raw A data frame containing a raw import of 'electric-vehicle-sales-by-region-and-scenario-2030-and-2035.xlsx'
+#' sheet : electric-vehicle-sales-by-regio
 #' @param mpp_ats_raw A tidyxl data frame containing a raw import of `2022-08-12
 #'   - MPP ATS - RPK and GHG intensity.xlsx`.
-#'
+#' @param iea_sales_share_ev A data frame containing a raw import of 'electric-vehicle-sales-by-region-and-scenario-2030-and-2035.xlsx'
+#' - sheet: electric vehicle share-ev
 #' @return A prepared WEO 2024 scenario data-frame.
 #'
 #' @importFrom dplyr %>%
 #'
 #' @export
-
-prepare_weo_2024_scenario <- function(weo_2024_ext_data_regions_raw,
+prepare_weo_2024_ecb_scenario <- function(weo_2024_ext_data_regions_raw,
                                       weo_2024_ext_data_world_raw,
                                       weo_2024_fig_chptr_3_raw,
-                                      iea_global_ev_raw,
+                                      iea_global_ev_2024_raw,
+                                      iea_sales_share_ev,
                                       mpp_ats_raw) {
-  weo_2024_automotive <- weo_2024_extract_automotive(weo_2024_fig_chptr_3_raw, iea_global_ev_raw)
+  weo_2024_ecb_automotive <- weo_2024_extract_automotive(
```

Comment:
where is this function defined? `weo_2024_extract_automotive()`

when I search for it, I get no results and that makes the workflow error out